src/corner_kick/src/components/LayersPanel/__stories__/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file creates a story for the LayersPanel
+ */
+
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+
+import { StorybookStage } from 'SRC/components/StorybookStage';
+import { LayersPanel } from '..';
+
+const stories = storiesOf('Layers Panel', module);
+
+/**
+ * Story describing a LayerPanel with some layers defined
+ */
+stories.add('with layers', () => (
+    <StorybookStage width="300px">
+        <LayersPanel
+            layers={[
```

Comment:
Just to clarify, these layers will later be dynamically created from the `Layer.msg` ROS messages? We won't need to hardcode layers right?

Reply:
Yup. This is just a visual test to make sure the UI is working correctly.

You can see how it looks here: https://deploy-preview-271--adoring-mestorf-e664ee.netlify.com/?selectedKind=Layers%20Panel&selectedStory=with%20layers&full=0&addons=1&stories=1&panelRight=0